### ISSUE TITLE
Explain data model in greater detail in Advanced Concepts section

### ISSUE THREAD

**Issue by msporny** [w3c/vc-data-model#268]
Add pretty pictures, add lots of prose to talk about graph containers, containment, etc.

**Comment by TzviyaSiegman** [comment]
I can write the image descriptions.

**Comment by msporny** [comment]
@TallTed -- I have tried to expand upon what a "Verifiable Credential" actually looks like in an attempt to address your concerns related to the oversimplification in [Section 3.2: Credentials](https://w3c.github.io/vc-data-model/#credentials). The idea is that this image below would be expanded upon in the "Syntactic Sugar" section OR at the end of the section 3.2.

https://docs.google.com/drawings/d/1v293yj2Z4J9JHDTrTjnLEBs_D1zQTMp1ocgR8KF2DgI/edit

Thoughts? Is this the direction you were hoping for?

**Comment by David-Chadwick** [comment]
I think this picture is very helpful. 

However, I do believe that it could be turned into a tree by having VerifiableCredential as the root, with arcs labelled Proof and Credential emanating from the root to your Signature 456 and Credential 123 nodes.

**Comment by David-Chadwick** [comment]
One minor point. The subject should not be Pat, but rather an ID.

**Comment by TallTed** [comment]
@msporny - Capturing roughly what was said in the concall...  This is the direction I was looking for.  Changing the diagram from focusing on the magical "ProofOfAgeCredential" to a university-related credential will help real-world understanding, and this should then help lead to production of some generalized diagram(s) applicable to any VC.

**Comment by msporny** [comment]
This image is in the spec now: https://docs.google.com/drawings/d/1v293yj2Z4J9JHDTrTjnLEBs_D1zQTMp1ocgR8KF2DgI/edit

There is one more image for presentation that I need to add. The data model has also been simplified to remove certain graph containers, and where we use them, we explicitly state it and explain it (we don't dumb it down anymore). So, one more image and I think we can close this issue.

**Comment by msporny** [comment]
Final image added for presentations in https://github.com/w3c/vc-data-model/commit/b608e721220fa7aae02f18173715359f27674b14. I believe that addresses all of the requests for more detail when introducing and describing credentials/presentations. Closing, re-open if you disagree that we're done here.
